### DISCUSSION TITLE
How to retain interjections within the transcription?

### DISCUSSION THREAD

**Discussion by chuuck** [openai/whisper#580] [Q&A] [answered]
Hi there,

When testing the Whisper against real world scenarios, it seems like Whisper automatically removes or either doesn't recognise interjections in speech. With the samples I have played around with it seems to either skip interjections or make up a word that doesn't even exist (further digging showed me that those tokens normally have low confidence score). Some examples of interjections are in this website [here](https://www.vidarholen.net/contents/interjections/). 

My goal is to include them within the transcription. Do you know if there is a solution possible that would allow including interjections?

Thank you!

**Comment by jianfch** [comment, marked as answer]
See #286
